src/plugin.js

```diff
@@ -13,37 +13,45 @@ const defaults = {
   errors: {
     '1': {
       type: 'MEDIA_ERR_ABORTED',
-      headline: 'The video download was cancelled'
+      headline: 'The video download was cancelled',
+      dismiss: true
```

Comment:
Good point. I'm was thinking whether specific default errors should not be able to be dismissed.